apps/litter-networker-api/src/services/bag_counts_service.py

```
@@ -4,12 +4,14 @@
 from __future__ import annotations
 
 import getpass
+import logging
 import math
 import time
 import uuid
 from dataclasses import dataclass
+from decimal import Decimal, ROUND_HALF_UP
 from datetime import datetime, timezone
-from typing import Any, Protocol
+from typing import Any, Callable, Protocol
```

Ruff (0.15.7)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)